src/app/components/SearchResultsPage/SearchResultsPage.jsx

```diff
@@ -80,6 +81,31 @@ class SearchResultsPage extends React.Component {
     return false;
   }
 
+  getSubjectLiteralFilters(apiFilters) {
```

Comment:
I think the three functions to find the selected subjectLiterals can be taken out and make them an independent module?

Reply:
OK, I'll do that. Not sure I understand your question above about the clearAllFilters button, but I think the ternary is unnecessary now. I will remove it.